pkg/river/internal/value/decode_test.go

```diff
@@ -650,31 +651,32 @@ func TestDecode_KnownTypes_Any(t *testing.T) {
 		input  any
 		expect any
 	}{
-		// All numbers must decode to float64.
-		{int(15), float64(15)},
-		{int8(15), float64(15)},
-		{int16(15), float64(15)},
-		{int32(15), float64(15)},
-		{int64(15), float64(15)},
-		{uint(15), float64(15)},
-		{uint8(15), float64(15)},
-		{uint16(15), float64(15)},
-		{uint32(15), float64(15)},
-		{uint64(15), float64(15)},
+		{int(15), int(15)},
+		{int8(15), int(15)},
+		{int16(15), int(15)},
+		{int32(15), int(15)},
+		{int64(15), int64(15)},
+		{uint(15), uint64(15)},
+		{uint8(15), uint64(15)},
+		{uint16(15), uint64(15)},
+		{uint32(15), uint64(15)},
+		{uint64(15), uint64(15)},
```

Comment:
All of these should fit within an `int`, so I'm not sure this is matching the behavior we were looking for.